src/alpine/3.13/WithNode/amd64/Dockerfile

```diff
@@ -1,34 +1,9 @@
 FROM mcr.microsoft.com/dotnet-buildtools/prereqs:alpine-3.13
-RUN apk update
 
-ENV NODE_VERSION=14.16.1
-
-### INSTALL NODE
-RUN apk add --no-cache nodejs
-
-ENV YARN_VERSION=1.10.1
-
-RUN apk add --no-cache --virtual .build-deps-yarn \
-        gnupg \
-        tar \
-        && for key in \
-            6A010C5166006599AA17F08146C2130DFD2497F5 \
-        ; do \
-            gpg --keyserver hkps://keyserver.ubuntu.com --recv-keys "$key" \
-            || gpg --keyserver hkps://keys.openpgp.org --recv-keys "$key"; \
-        done \
-            && curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz" \
-            && curl -fsSLO --compressed "https://yarnpkg.com/downloads/$YARN_VERSION/yarn-v$YARN_VERSION.tar.gz.asc" \
-            && gpg --batch --verify yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-            && mkdir -p /opt \
-            && tar -xzf yarn-v$YARN_VERSION.tar.gz -C /opt/ \
-            && ln -s /opt/yarn-v$YARN_VERSION/bin/yarn /usr/local/bin/yarn \
-            && ln -s /opt/yarn-v$YARN_VERSION/bin/yarnpkg /usr/local/bin/yarnpkg \
-            && rm yarn-v$YARN_VERSION.tar.gz.asc yarn-v$YARN_VERSION.tar.gz \
-            && apk del .build-deps-yarn
+RUN apk update && apk add --no-cache nodejs yarn
```

Comment:
Recommended to list out each package name on its own line.

Reply:
Updated.